aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -366,6 +366,31 @@ Tensor& logical_not_out(Tensor& result, const Tensor& self) {
   return result;
 }
 
+Tensor& signbit_out(Tensor& result, const Tensor& self) {
+  TORCH_CHECK(!self.is_complex(), "clamp is not yet implemented for complex tensors.");
```

Comment:
It's true that clamp is not implemented for complex tensors, but this should probably say:

"signbit is not implemented for complex tensors."

No need for a "yet," either, since we have no plans to implement it in the future -- it's not even clear how to define a function like signbit on complex values. NumPy, for example, also throws an error when signbit is given a complex input.

Reply:
Also: better add a check that result is bool

Reply:
Addressed! Thanks for the correction.